test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Your have added your test.md file in .gitignore file but it is still visible in your Github repository because you have added it in the project before creating your .gitignore file. To fix this follow these steps :memo:

1. **Remove your test.md file from your repo** :yellow_circle:
2. **Add it again in your local repo to check if .gitignore is ignoring this file** :thinking:
3. **Make a commit and push** :+1: